dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```
@@ -43,7 +43,11 @@ LABEL \
   org.label-schema.vendor="{{ .BeatVendor }}" \
   org.label-schema.license="{{ .License }}" \
   org.label-schema.name="{{ .BeatName }}" \
+{{- if .Snapshot }}
+  org.label-schema.version="{{ beat_version }}-SNAPSHOT" \
+{{- else }}
   org.label-schema.version="{{ beat_version }}" \
+{{- end }}
```

Nit. In other places we do it like this:
```suggestion
  org.label-schema.version="{{ beat_version }}{{if .Snapshot}}-SNAPSHOT{{end}}" \
```

What I find slightly better (apart of aesthetic opinions), to have a single place with the label name.